terra2/assetlist.json

```diff
@@ -366,6 +366,33 @@
         }
       ]
     },
+    {
+      "description": "ClonEliteFund invests in Cosmos ecosystem projects, focusing on scalability, security, and interoperability to maximize returns.",
```

Comment:
Please relocate this asset to the bottom of the list.

Reply:
sir, is it really necessary, I see that the code layout in other entries is the same as mine, tell me where exactly I should move it or please do it if you can

Reply:
the layout of properties within the asset was fine, but new assets always go to the bottom of the list--you just inserted this somewhere in the middle of the list